src/gui/editors/AutomationEditor.cpp

```diff
@@ -2107,7 +2107,7 @@ AutomationEditorWindow::AutomationEditorWindow() :
 
 	for( float const & zoomLevel : m_editor->m_zoomXLevels )
 	{
-		m_editor->m_zoomingXModel.addItem( QString( "%1\%" ).arg( zoomLevel * 100 ) );
```

Comment:
Just to understand, why were there complaints? Because the slash is unnecessary? Would it be correct to use `%1%%` here?

Reply:
It's because `\%` is not a valid escape sequence. You can see the valid ones on the wiki here: https://en.cppreference.com/w/cpp/language/escape. I expect (but haven't checked) that the double percent sign in `%1%%` would end up in the resulting string, e.g. "100%%". [`QString::arg`](https://doc.qt.io/qt-5/qstring.html#arg) doesn't say it replaces anything other than `%<n>` sequences, and I wouldn't expect it to touch or require `%%` as it's designed to be chained several times in a row (otherwise the first call would break things for subsequent calls).

Reply:
Hmm, yes, just tested that it ends up in "100%%". Then I get it. Resolving the comment.